packages/sit-onyx/src/components/OnyxLink/OnyxLink.vue

```diff
@@ -29,15 +33,49 @@ defineSlots<{
     @click="emit('click')"
   >
     <slot></slot>
+    <OnyxIcon
+      v-if="!props.hideExternalIcon && isExternalLink(props.href)"
```

Comment:
We need to clarify (and/or research) if the icon means "opens external page in the same tab" (when `target`) or "opens external page in the same or new tab" (by checking the origin). What is the common/recommended pattern? I have seen both in the past.

Reply:
We need to document in the description what an external link means.

Reply:
Done